src/NHSD.BuyingCatalogue.Solutions.API/ViewModels/Solution/ImplementationTimescalesSectionAnswers.cs

```diff
@@ -0,0 +1,18 @@
+using Newtonsoft.Json;
+using NHSD.BuyingCatalogue.Solutions.Contracts;
+
+namespace NHSD.BuyingCatalogue.Solutions.API.ViewModels.Solution
+{
+    public sealed class ImplementationTimescalesSectionAnswers
+    {
+        [JsonProperty("description")]
+        public string Description { get; }
+
+        [JsonIgnore] public bool HasData => !string.IsNullOrWhiteSpace(Description);
```

Comment:
Formatting... Could the property go under the JsonIgnore attribute?

Reply:
ah yeah I don't know how this happened :)